services/dex-k8s-authenticator/1.4.2/dex-k8s-authenticator.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 1.3.2
+      version: 1.4.1
```

Comment:
This version is correct, as chart version was bumped to `1.4.1` - https://github.com/mesosphere/charts/blob/master/staging/dex-k8s-authenticator/Chart.yaml#L5